### PR TITLE
Added changes according to Sergey's comments

### PR DESCRIPTION
**Pull request by BOksanaV**
Added changes according to Sergey's comments and added Thread.sleep(2000) for test stability ;

### CODE REVIEW

File: src/test/java/GroupBugHuntersTest.java [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
+
         WebElement searchBox = getDriver().findElement(By.xpath("//input[@name='search_query']"));
-        WebElement searchButton = getDriver().findElement(By.xpath("//button[@type ='submit']"));
         searchBox.sendKeys("soap flower bouquet");
+        Thread.sleep(2000);
+
+        WebElement searchButton = getDriver().findElement(By.xpath("//button[@type ='submit']"));
         searchButton.click();
+
         searchBox = getDriver().findElement(By.xpath("//input[@name='search_query']"));
```
Comment (anonymoustuna) [inline src/test/java/GroupBugHuntersTest.java:21]: You already initialized searchBox as WebElement above and might no need in this line.

File: src/test/java/GroupBugHuntersTest.java [hunk 2]
```diff
@@ -7,13 +7,17 @@
 public class GroupBugHuntersTest extends BaseTest {
 
     @Test
-    public void testOksanaBakatova()  {
+    public void testOksanaBakatova() throws InterruptedException {
```
Comment (anonymoustuna) [inline src/test/java/GroupBugHuntersTest.java:11]: You might get rid of this space.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/GroupBugHuntersTest.java (hunk 1 above)
Comment (BOksanaV) [reply src/test/java/GroupBugHuntersTest.java:20]: Test didn't work without this line. Thank you.

File: src/test/java/GroupBugHuntersTest.java (hunk 2 above)
Comment (BOksanaV) [reply src/test/java/GroupBugHuntersTest.java]: done

File: src/test/java/GroupBugHuntersTest.java [hunk 3]
```diff
@@ -7,13 +7,16 @@
...
-
+    public void testOksanaBakatova() throws InterruptedException {
         getDriver().get("https://www.etsy.com");
+
         WebElement searchBox = getDriver().findElement(By.xpath("//input[@name='search_query']"));
-        WebElement searchButton = getDriver().findElement(By.xpath("//button[@type ='submit']"));
         searchBox.sendKeys("soap flower bouquet");
+        Thread.sleep(2000);
+
+        WebElement searchButton = getDriver().findElement(By.xpath("//button[@type ='submit']"));
```
Comment (ViktorPodgornov) [inline src/test/java/GroupBugHuntersTest.java:17]: You used "searchButtom" one time. You can not create the variable.
getDriver().findElement(By.xpath("//**button**[@type ='**submit**']")).click();

File: src/test/java/GroupBugHuntersTest.java (hunk 1 above)
Comment (ViktorPodgornov) [reply src/test/java/GroupBugHuntersTest.java:20]: By the way, I could be wrong, but in my opinion, Sergey said that overwriting variables in tests is a bad practice.

**Review by ViktorPodgornov — APPROVED** [review]
I approve, but I think the problem was not in the "sleep". Most likely the problem with the site. Since after multiple launches, sometimes the test does not enter the search query and failed.

**Pull request merged**